vim/autoload/himalaya/mbox.vim

```diff
@@ -25,27 +25,44 @@ function! himalaya#mbox#curr_mbox()
   return s:curr_mbox
 endfunction
 
+function! s:telescope_picker(mboxes)
+  call luaeval('require("himalaya.mbox").mbox_picker')(a:mboxes)
+endfunction
+
+function! s:fzf_picker(mboxes)
+  call fzf#run({
+    \"source": a:mboxes,
+    \"sink": function("himalaya#mbox#post_input"),
+    \"down": "25%",
+  \})
+endfunction
+
+function! s:input_picker(mboxes)
```

Comment:
`input` is maybe to vague (now that we have more pickers available), what do you think about `native_picker`?

Reply:
Changed